### PR TITLE
Fix other areas tutorial.

### LINKED ISSUE #629: UK example - stack trace
Hi,

If I follow the documented example for other regions (Shrewsbury in the UK) , 

[https://github.com/synthetichealth/synthea/wiki/Other-Areas](https://github.com/synthetichealth/synthea/wiki/Other-Areas)

setting the contents of the zipfile, demographics and hospital csv files as shown 

&  generate.geography.country_code = UK

Then:

run-synthea -s 0 -p 1 "West Midlands" Shrewsbury

I get a stack trace:

java.lang.NullPointerException
        at org.mitre.synthea.world.agents.Person.setProvider(Person.java:529)
        at org.mitre.synthea.world.agents.Person.setProvider(Person.java:542)
        at org.mitre.synthea.world.agents.Person.getProvider(Person.java:520)
        at org.mitre.synthea.world.agents.Person.encounterStart(Person.java:456)
        at org.mitre.synthea.modules.EncounterModule.createEncounter(EncounterModule.java:133)
        at org.mitre.synthea.modules.EncounterModule.process(EncounterModule.java:64)
        at org.mitre.synthea.engine.Generator.generatePerson(Generator.java:347)
        at org.mitre.synthea.engine.Generator.lambda$run$2(Generator.java:259)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)

I also get the same when trying to use my own UK data set as well.

Any hints on how to workaround/fix please ?

Does anyone else get the same ?

### PR DESCRIPTION
**Pull request by jawalonoski**
Fixes #629 and #640 .

Also, I already changed the [Other Areas](https://github.com/synthetichealth/synthea/wiki/Other-Areas) wiki entry to correspond to the changes in this branch, so until this branch is merged into `master` the wiki instructions will not work on `master` (the wiki instructions were already broken, so I do not see this as a big deal).

### CODE REVIEW

**Comment by jawalonoski** [comment]
I am going to delete the `src/main/resources/**/*_uk.csv` files before I merge this into `master`. They are only temporary.

File: src/test/java/org/mitre/synthea/world/agents/PayerTest.java [hunk 1]
```diff
@@ -25,29 +28,27 @@
 
 public class PayerTest {
 
+  static String testState;
```
Comment (eedrummer) [inline src/test/java/org/mitre/synthea/world/agents/PayerTest.java]: I know it's just a test and that it was this way previously, but default visibility is rarely used in Java.

File: src/main/resources/synthea.properties [hunk 2]
```diff
@@ -63,10 +63,10 @@ generate.database_type = none
 # none = no database, limits certain features but increases throughput
 
 # default demographics is every city in the US
-generate.demographics.default_file = geography/demographics.csv
-generate.geography.zipcodes.default_file = geography/zipcodes.csv
-generate.geography.country_code = US
-generate.geography.timezones.default_file = geography/timezones.csv
+generate.demographics.default_file = geography/demographics_uk.csv
```
[the remark this answers is not in the document]
Comment (jawalonoski) [reply src/main/resources/synthea.properties]: No, this is temporary. I'm going to change it back and remove the UK files. They are now in https://github.com/synthetichealth/synthea-international

[the remark this answers is not in the document]
Comment (jawalonoski) [reply src/main/resources/synthea.properties]: Reverted.

File: src/test/java/org/mitre/synthea/export/ExporterTest.java [hunk 3]
```diff
@@ -68,7 +74,8 @@
     assertEquals(128, encounter.observations.get(0).value);
   }
 
-  @Test public void test_export_filter_should_keep_old_active_medication() {
+  @Test
+  public void test_export_filter_should_keep_old_active_medication() {
```
[the remark this answers is not in the document]
Comment (jawalonoski) [reply src/test/java/org/mitre/synthea/export/ExporterTest.java]: Fixed.

File: src/test/java/org/mitre/synthea/world/agents/PayerTest.java (hunk 1 above)
Comment (jawalonoski) [reply src/test/java/org/mitre/synthea/world/agents/PayerTest.java]: Fixed.

**Pull request merged**